SConstruct

```diff
@@ -706,20 +706,14 @@ if selected_platform in platform_list:
             suffix="glsl.gen.h",
             src_suffix=".glsl",
         ),
+        "GLES3_GLSL": env.Builder(
+            action=run_in_subprocess(gles3_builders.build_gles3_headers),
```

Comment:
This should be changed to a `Run` command as done in d86de6c98e435d31bfdebc50d2db6d4d4048be40.